tests/tabs/compositionComponent.test.jsx

```diff
@@ -15,12 +15,6 @@ import sinon from 'sinon';
 import CompositionComponent from '../../app/js/components/tabs/tabcomponents/compositionComponent';
 
 describe('<CompositionComponent />', ()=>{
-    it('should mount the compositionComponent in the dom', ()=>{
```

Comment:
Is there a reason why you remove the test?

Reply:
Yes. It's not needed, as can be seen on line 30 on `compositionComponent.jsx`, componentDidMount was placed just so this test could be written. The other test passing shows that the component was actually mounted as the div elements won't be available if the component is not mounted

Reply:
Alright cool. 👍